BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -423,6 +432,13 @@ public final class BlueprintView: UIView {
         return rootController.children
     }
 
+    /// Forces a synchronous layout, for testing purposes.
+    @_spi(Debugging)
```

Comment:
Ah, hm, I wonder if we should/could drop the contextual prefixes from our SPI annotations without causing conflicts (e.g. you have two frameworks with a `Debugging` SPI namespace that you want to import in a single file).

Reply:
I personally prefer the prefix of `Market`, `Blueprint`, etc since it's easier to cmd + f for all the spis for a single module, as well.

Reply:
> Ah, hm, I wonder if we should/could drop the contextual prefixes from our SPI annotations without causing conflicts (e.g. you have two frameworks with a `Debugging` SPI namespace that you want to import in a single file).

From a technical perspective that shouldn't be a problem, since these are scoped to each import.

I don't have a strong preference on naming, since this will be used in like 2 places. ¯\_(ツ)_/¯